commands/auth.go

```diff
@@ -127,10 +136,32 @@ func loginCmdF(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+	passwordFile, _ := cmd.Flags().GetString("password-file")
+	if password != "" && passwordFile != "" {
+		return errors.New("can not use two passwords at the same time")
```

Comment:
```suggestion
		return errors.New("cannot use two passwords at the same time")
```

Same for the other occurrences.